contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -32,6 +32,7 @@
 #include "parser/parse_relation.h"
 #include "parser/parse_target.h"
 #include "parser/parse_relation.h"
+#include "parser/scansup.h"  /* downcase_identifier */
```

Comment:
We can remove this header. We are not using downcase_identifier in the function.